server/src/main/java/org/elasticsearch/common/logging/RateLimitingFilter.java

```diff
@@ -68,7 +69,7 @@ public Result filter(Message message) {
             final ESLogMessage esLogMessage = (ESLogMessage) message;
 
             String xOpaqueId = esLogMessage.get(X_OPAQUE_ID_FIELD_NAME);
-            final String key = esLogMessage.get("key");
+            final String key = esLogMessage.get(KEY_FIELD_NAME);
```

Comment:
as per comment above, do you think it would be better to use `DeprecatedMessage.KEY_FIELD_NAME` and `DeprecatedMessage.X_OPAQUE_ID_FIELD_NAME` ?